fastlane/Pluginfile

```diff
@@ -3,7 +3,7 @@
 # Ensure this file is checked in to source control!
 
 group :screenshots, optional: true do
-  gem 'rmagick', '~> 3.2.0'
+  gem 'rmagick', '~> 4.1'
 end
 
-gem 'fastlane-plugin-wpmreleasetoolkit', git: 'https://github.com/wordpress-mobile/release-toolkit', tag: '0.9.11'
+gem 'fastlane-plugin-wpmreleasetoolkit', git: 'https://github.com/AliSoftware/release-toolkit', branch: 'fix/rmagick-deprecated-constant'
```

Comment:
This needs to be reverted before undrafting that PR, once https://github.com/wordpress-mobile/release-toolkit/pull/154 is merged and release-toolkit got released, to instead point to the future new version/tag of release-toolkit

Reply:
`release-toolkit` 0.9.13 has been released and ships the changes in the mentioned PR.